doc/use_cases.md

```diff
@@ -0,0 +1,49 @@
+# Use case scenarios for posteriordb
+
+posteriordb has different uses. Below is list of example use cases we have been thinking and what features are important in each use case
+
+## Testing implementations of inference algorithms with asymptotically decreasing bias and variance
+
+If an inference algorithm has asymptotically decreasing bias and variance, then an implementation should have decreasing error compared to true posterior summaries.
+
+True summaries can be computed analytically for some simpler models, which are not necessarily that interesting from the modelling perspective, but are useful for detecting problems as the truth is known exactly. It is easiest to compare estimated and true posterior moments and quantiles for scalar quantities. For closed form marginals we can also use QQ-plots and ECDF-plots for visual exploration of unexpected discrepancies. Uniformity of PIT tests can be used to test the whole marginal, although the interpretation of different uniformity tests is more complex than point estimate comparisons.
```

Comment:
I think we should define abbreviations the first time we use them (PIT, QQ, ECDF and MCMC). To most people, most abbreviations are known, but if a person doesn't know, they will search for the term and then having the full term will help them find information quicker.

Reply:
sure